content/app/development/ux/styling/_index.en.md

```diff
@@ -165,6 +165,37 @@ Det vil se omtrent slik ut
 
 ![labelGrid eksempel output](label-grid.png "labelGrid eksempel output")
 
+## To radioknappalternativer eller avkrysningsbokser under hverandre
+Som standard kommer radioknapper eller checkboxer som har kun to svaralternativer, på én linje. Dersom man ønsker at svarene skal ligge på hver sin linje slik:
+
+![SvaralternativUnderHverandre](https://user-images.githubusercontent.com/42466346/177565112-ff5313f7-f56d-4421-bd17-b53f69308697.jpg)
```

Comment:
Please upload this image to the repository and show that image instead. Like the labelGrid example on line 166.
This will make the docs more self contained.